playground/assets/__tests__/assets.spec.ts

```diff
@@ -145,27 +145,20 @@ describe('css url() references', () => {
-    )
+    expect(imageSet).toContain('image-set(url("data:image/png;base64,')
```

Comment:
The updated chromium browser now supports [image-set](https://developer.mozilla.org/en-US/docs/Web/CSS/image/image-set#browser_compatibility), so it's returning a different computed value here that needs to be updated.